src/sugar3/graphics/xocolor.py

```diff
@@ -224,7 +231,12 @@ def _parse_string(color_string):
 
 
 class XoColor:
-
+    '''
+    Defines color for XO
+    
+    Args:
+        color_string (string): 
+    '''
```

Comment:
Please document...

* That None results in the user's colour being created
* That the colour string should be 2 hex colours, comma seperated (eg. `#FFFFF,#111111`), "white" or "insensitive"
* That a random colour combo will be used if it can not be parsed